impeller/entity/shaders/glyph_atlas.frag

```diff
@@ -7,29 +7,19 @@
 uniform sampler2D glyph_atlas_sampler;
 
 uniform FragInfo {
-  vec2 atlas_size;
   vec4 text_color;
 }
 frag_info;
 
-in vec2 v_unit_position;
-in vec2 v_source_position;
-in vec2 v_source_glyph_size;
+in vec2 v_uv;
 in float v_has_color;
 
 out vec4 frag_color;
 
 void main() {
-  vec2 uv_size = v_source_glyph_size / frag_info.atlas_size;
-  vec2 uv_position = v_source_position / frag_info.atlas_size;
-  if (v_has_color == 1.0) {
-    frag_color =
-        texture(glyph_atlas_sampler, v_unit_position * uv_size + uv_position) *
-        frag_info.text_color.a;
-  } else {
-    frag_color =
-        texture(glyph_atlas_sampler, v_unit_position * uv_size + uv_position)
-            .aaaa *
-        frag_info.text_color;
+  vec4 value = texture(glyph_atlas_sampler, v_uv);
+  if (v_has_color != 1.0) {
```

Comment:
need to get rid of this too

Reply:
Perhaps this wouldn't be an issue if we just had separate atlases. Leaving for another day.